components/eamxx/src/share/io/scorpio_output.cpp

```diff
@@ -331,7 +400,7 @@ void AtmosphereOutput::run (const std::string& filename, const bool is_write_ste
 long long AtmosphereOutput::
 res_dep_memory_footprint () const {
   long long rdmf = 0;
-  if (m_remapper) {
+  if (m_horiz_remapper) {
```

Comment:
The vertical remapper also contributes to memory footprint, no? Perhaps we could just do
```
if (m_io_field_mgr != m_sim_field_mgr)
```